backend/Board.py

```diff
@@ -1,11 +1,42 @@
 # import pygame as pg
-from backend.Cell import Cell, CellType
+from backend.Cell import Cell, Color
 
 
 class Board:
     def __init__(self, size):
         self.board_size = size
         self.create_initial_board(size)
+        self.adjacent_positions_map = {
```

Comment:
We definitely cant have this. Need a programmatic way of determining adjacent piece so that we can introduce new boards easily. 

Valid moves are left, right, up, down for any given clicked piece. Some of these option aren't available due to index not existing or piece already being there.

Reply:
From previous project [Add moves function](https://github.com/CS-449-Group-Project/NineMensMorris/blob/master/src/Morris_FX/Logic/GameManager.java#L402-L426)

Reply:
I very much agree.

Reply:
I don't see any problem of doing this way, if we somehow do it in a programmatic way it will be prone to errors, even if it does not, mentioning coodinates like this is fast and eazy way to implement.

i know you did in your last project differently but i cannot see any benefit of doing that way. also this approach is indeed faster, for new boards we can maintain seperate file if you want that can be changed if we want to mess with coordinates

i know in project we avoid hardcoding but this is kind of static and don't change very often